src/sass/media-queries.scss

```diff
@@ -1,26 +0,0 @@
-@media screen and (max-width: 768px) {
- #page-wrapper {
-      #content {
-          .page-section {
-              & > .tab-menu {
-                  ul.tm-ul {
-                      li {
-                          &.active {
-                              a {
-                                  background: none;
-                              }
-                          }
-                      }
-                  }
-              }
-          }
-      }
-  }
-
-  #sell_popup_container {
```

Comment:
Yeah, none of the styles are needed but I have forgotten to include the non-media-query #sell_popup_container style. Will redo the PR, but apart from that, these are not needed.